evaluate/task/task-transpile_test.go

```diff
@@ -546,3 +476,114 @@ func TestValidateTranspileRepository(t *testing.T) {
 		})
 	})
 }
+
+func TestValidateImplementationDirectory(t *testing.T) {
```

Comment:
Bit confusing that we now have both `TestValidateTranspileRepository` and `TestValidateImplementationDirectory`. Can you add a docstring to both that the validation is split for simplicity and hence also the tests are split?

Reply:
Since the `implementation` folder structure changed (it only contains source files now), I removed the function that validates the `implementation` folder. 

Obsolete